### PR TITLE
Androidx compatability & update dagger version & build.gradle systax

### PR DESCRIPTION
**Pull request by hoangvunam**
Androidx compatability & update dagger version & build.gradle systax

### CODE REVIEW

File: MVPFragment/root/src/app_package/classes/Fragment.java.ftl [hunk 1]
```diff
@@ -1,12 +1,13 @@
...
 import android.os.Bundle;
-import android.support.annotation.NonNull;
-import android.support.annotation.Nullable;
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
 import android.view.LayoutInflater;
 import android.view.View;
 import android.view.ViewGroup;
 
+
```
Comment (benoitletondor) [inline MVPFragment/root/src/app_package/classes/Fragment.java.ftl]: I don't think this extra line break is needed

**Review by benoitletondor — CHANGES_REQUESTED** [review]
Hey there,

Thank you so much for your amazing work. It looks all good except that tiny line break. If you can fix that I'll be able to merge it :) 

Thank you again

File: MVPFragment/root/src/app_package/classes/Fragment.java.ftl (hunk 1 above)
Comment (hoangvunam) [reply MVPFragment/root/src/app_package/classes/Fragment.java.ftl]: I deleted this line, please check.

**Comment by benoitletondor** [comment]
Thank you!

**Pull request merged**